acra-censor/common/logging_logic.go

```diff
@@ -155,7 +162,9 @@ func (queryWriter *QueryWriter) WriteQuery(query string) {
 	case queryWriter.signalWriteQuery <- query:
```

Comment:
(very minor thing) there's a redundant space in line above, couldn't comment that exact line since it was not part of the committed diff

Reply:
hm... I don't understand which tab you mean. Plus all code passed through gofmt that removes extra tabs and tests verify that gofmt used for code.

Reply:
Seems like it was rendered differently for some reason on my side